akka-persistence-testkit/src/test/scala/akka/persistence/testkit/javadsl/CommonTestkitTests.scala

```diff
@@ -0,0 +1,575 @@
+/*
+ * Copyright (C) 2018-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.testkit.javadsl
+
+import akka.actor.Props
+import akka.persistence._
+import akka.persistence.testkit._
+import akka.testkit.EventFilter
+import org.scalatest.Matchers._
+import akka.util.ccompat.JavaConverters._
+import akka.actor.typed.javadsl.Adapter
+
+trait CommonTestkitTests extends JavaDslUtils {
```

Comment:
More consistent to spell `TestKit` with a capital `K` in class/test names

Reply:
corrected